afkak/_util.py

```diff
@@ -94,7 +104,7 @@ def write_short_ascii(s):
     """
     if s is None:
         return _NULL_SHORT_STRING
-    if not isinstance(s, string_types):
+    if not isinstance(s, (str, bytes)):
```

Comment:
Doc string should be updated to Python 3 only, and to mention bytes.

Reply:
It looks like accepting `bytes` here was a bug: it should only be `str` on Python 3.